packages/git/git.3.3.2/opam

```diff
@@ -30,9 +30,9 @@ depends: [
   "mimic"
   "cstruct" {>= "5.0.0"}
   "angstrom" {>= "0.14.0"}
-  "carton" {>= "0.4.0"}
-  "carton-lwt" {>= "0.4.0"}
-  "carton-git" {>= "0.4.0"}
+  "carton" {= "0.4.0"}
+  "carton-lwt" {= "0.4.0"}
+  "carton-git" {= "0.4.0"}
```

Comment:
could you use `>= "0.4.0" & < "0.4.1"` instead? Just in case for whatever reason there is a revision between the two